### PR TITLE
UN-387/471 Intro text missing after change to field name

### PR DESCRIPTION
**Pull request by jamesbiggs**
Ticket URL: https://national-archives.atlassian.net/browse/UN-387

## About these changes

Back-end change to standfirst/intro text meant that the front-end intro text wasn't being displayed. Changed to match this.

## How to check these changes

Check a record revealed page and ensure that the intro text is being displayed on the front-end

## Before assigning to reviewer, please make sure you have

- [x] Checked things thoroughly before handing over to reviewer.
- [x] Checked PR title starts with ticket number as per project conventions to help us keep track of changes.
- [x] Ensured that PR includes only commits relevant to the ticket.
- [x] Waited for all CI jobs to pass before requesting a review.
- [x] Added/updated tests and documentation where relevant.

## Merging a feature or bug fix?

1. Use the `Squash and merge` option to keep the target branch's commit history nice and clean.
2. Where relevant, include the ticket number in commit title, e.g. `DF-XXX: Ticket name / short description`.

## Merging a release branch?

1. Use the `Create a merge commit` option to preserve the original commit IDs.
2. Use the message format `release/<major.minor.patch>` when merging.

## Deployment guidance

1. Post to the appropriate Slack channel to check that it is okay to continue.
2. To start a deployment, push the relevant branch to the `platform` remote.
3. Update the appropriate Slack channel when the deployment is complete.
4. Update the status on the corresponding Jira ticket (where relevant).

### CODE REVIEW

**Review by William-Blackie — APPROVED** [review]
Looks good to me @jamesbiggs. Nice work.

**Comment by ababic** [comment]
Hold on :)

File: templates/includes/record-revealed.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
             <div class="col-md-8 col-sm-12">
                 <p class="explore-intro__title explore-intro__title--meta">Record revealed</p>
                 <h1 class="explore-intro__title">{{ page.title }}</h1>
-                <p class="explore-intro__paragraph">{{ page.standfirst }}</p>
+                <p class="explore-intro__paragraph">{{ page.intro }}</p>
```
Comment (ababic) [inline templates/includes/record-revealed.html:8]: `page.intro` is RichText, for should have the `|richtext` filter (from `wagtailcore_tags`) applied to it. 

We'll also need to lose the hardcoded `<p>` tag, as the generated HTML will have it's own.

@William-Blackie What are the implications here on styling?

**Review by ababic — CHANGES_REQUESTED** [review]

File: templates/includes/record-revealed.html (hunk 1 above)
Comment (jamesbiggs) [reply templates/includes/record-revealed.html]: Hi @ababic this should be done now! 👍

**Comment by jamesbiggs** [comment]
Closing as Will is taking this up in his PR for UN-496

**Pull request closed without merging**